pyrogram/client/methods/messages/send_poll.py

```diff
@@ -110,7 +110,7 @@ def send_poll(
                         public_voters=not is_anonymous or None,
                         quiz=type == "quiz" or None
                     ),
-                    correct_answers=[bytes([correct_option_id])] if correct_option_id else None
+                    correct_answers=[bytes([correct_option_id])] if not correct_option_id == None else None
```

Comment:
I think would be better if you use this:
```python
correct_answers=None if correct_option_id is None else [bytes([correct_option_id])]
```
Both codes should work as expected, but the above one is more consistent (The same style is used on other parts of the library).